src/core/src/op/paged_attention.cpp

```diff
@@ -177,7 +177,8 @@ void PagedAttentionExtension::validate_and_infer_types() {
             get_input_partial_shape(15).rank().get_length(),
             ".");
         NODE_VALIDATION_CHECK(this,
-                              get_input_element_type(15).is_dynamic() || get_input_element_type(15) == element::f32,
+                              get_input_element_type(15).is_dynamic() || get_input_element_type(15) == element::f32 ||
+                                  get_input_element_type(15) == element::f16,
                               "Element type of `rotation_trig_lut` input should be f32, but it is ",
```

Comment:
minor: please fix the message, need to add mention of fp16

Reply:
fixed along with https://github.com/openvinotoolkit/openvino/pull/28429